hat/assets/js/apps/Iaso/domains/instances/compare/components/InstanceLogContentBasic.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { FunctionComponent } from 'react';
+import {
+    makeStyles,
+    Table,
+    TableBody,
+    TableCell,
+    TableRow,
+    TableHead,
+} from '@material-ui/core';
+import { useSafeIntl } from 'bluesquare-components';
+import { FileContent, FormDescriptor } from '../../types/instance';
+import { IntlFormatMessage } from '../../../../types/intl';
+import MESSAGES from '../messages';
+
+type Props = {
+    fileContent: FileContent;
+    fileDescriptor: FormDescriptor;
+};
+
+const styles = theme => ({
+    tableCellHead: {
+        fontWeight: 'bold',
+        backgroundColor: 'transparent',
+        borderTop: 'none !important',
+        borderLeft: 'none !important',
+        borderRight: 'none !important',
+        borderBottom: `1px solid ${theme.palette.ligthGray.border}  !important`,
+    },
+    tableCell: {
+        backgroundColor: 'transparent',
+        borderTop: 'none !important',
+        borderLeft: 'none !important',
+        borderRight: 'none !important',
+        borderBottom: `1px solid ${theme.palette.ligthGray.border}  !important`,
+    },
+});
+
+const useStyles = makeStyles(styles);
+
+const getLabelFromKey = (descriptor, key) => {
+    const field = descriptor.find(child => child.name === key);
+
+    // TO DO: find an efficient way to get label from group type fields (questions, meta, ...)
```

Comment:
May be have a look for `localizeLabel` in `hat/assets/js/apps/Iaso/domains/instances/utils.js`